_datascience/deidentification.md

```diff
@@ -1,114 +1,173 @@
+  6. Fax numbers
+  7. Device identifiers and serial numbers
+  8. Email addresses
+  9. Web Universal Resource Locators (URLs)
+  10. Social security numbers
+  11. Internet Protocol (IP) addresses
+  12. Medical record numbers
+  13. Biometric identifiers, including finger and voice prints
+  14. Health plan beneficiary numbers
+  15. Full-face photographs and any comparable images
+  16. Account numbers
+  17. Any other unique identifying number, characteristic, or code
+  18. Certificate/license numbers
+
+Additionally, the remaining information can not be used, either alone or in
+combination with other information, to identify an individual in the data set.
+More [guidance on using the safe harbor method for
+de-identification](https://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/index.html#safeharborguidance)
+is on the HHS website.
+
+## Limited datasets
+
+Similar to a de-identified data set, a [limited data
+set](https://www.hipaajournal.com/limited-data-set-under-hipaa/) excludes
+specific direct identifiers. However, a limited data set may include city,
+state, ZIP code, elements of a date (e.g. date of birth), and other
+characteristics or codes not listed as direct identifiers. Limited data sets may
+only be used for purposes of research, public health, and health care
+operations; and they are governed by a data use agreement. Unlike de-identified
+data sets, limited data sets are still considered to be PHI because they may
+contain identifiable information. 
+
+## De-identification of biospecimens
+
+The HIPAA Privacy Rule affects biospecimen resources when human specimens are
+accompanied by PHI. The Common Rule outlines when research studies involving
+biospecimens are considered human subjects research. Any research study where an
+"investigator obtains information or biospecimens through intervention or
+interaction with the individual, and uses, studies, or analyzes the information
+or biospecimens," is considered human subjects research. Only secondary research
+involving coded private information and coded biospecimens is considered not
+human subjects research. See the Office of Human Research Protection's [guidance
+on biospecimens use in
+research](https://www.hhs.gov/ohrp/coded-private-information-or-biospecimens-used-research.html)
+for more information on when biospecimens are considered coded or
+nonidentifiable.
+
+## Ethical considerations for use of de-identified data and specimens
+
+The story of Henrietta Lacks, her descendants, and the creation of HeLa cells,
+acted as a catalyst for proposed policy changes around biospecimen use and
+started a broader public discussion around the ethics of genomic research,
+especially secondary research with specimens.[^2]<sup>, </sup> [^3] Recent
+studies have confirmed that genomic data can never be truly annoynmized, and
+that it may be possible to re-identify seemingly de-identified data.[^2]<sup>,
```

Comment:
maybe add a note about how this risk needs to be explained in the consenting process